ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -37,15 +37,21 @@ component:
   heater: "Component which provides heat to media."
   humidifier: "Component which humidifies."
   pump: "Component used for the distribution of liquid media."
+  rack: "Component used in comms rooms for mounting servers & switches" # Added 25/08/20
   shade: "Window covering."
   tank : "Storage device for media."
   valve: "Component which meters the flow of water within a system or device."
   vane: "Component for guiding media flow."
   wheel: "Component used for transfer of heat from incoming to outgoing air streams. "
 descriptor: 
+  duty: "used to describe device used in duty/standby arrangement" # Added 27/08/20
+  standby: "used to describe device used in duty/standby arrangement" # Added 27/08/20
+  runtime: "accumulation of time a device or system has been running (quite often used for maintenance/servicing purposes)" # Added 18/08/20
+  condensate: "descriptor for pump used to remove liquid collected by condensation"  # Added 18/08/20
```

Comment:
this isn't only tied to pumps. something more direct like: 'the liquid collected through a process of condensation (e.g. condensate water drain pan, condensate water pump)